### DISCUSSION TITLE
v3.0.0-rc1

### DISCUSSION THREAD

**Discussion by gkoh** [gkoh/furble#144] [Announcements]
## Summary
* Complete UI rewrite with LVGL
   * Support for hardware button navigation
   * Support for touch screen
   * M5ez removed
* Upgrade to recent underlying bluetooth library
* Support for 'FauxNY', a virtual target camera to aid development

Install URL: https://857b8ef7.furble-web-installer.pages.dev/

## What's Changed
* Migrate UI to LVGL by @gkoh in https://github.com/gkoh/furble/pull/143

**Full Changelog**: https://github.com/gkoh/furble/compare/v2.3.0...v3.0.0-rc1

<hr /><em>This discussion was created from the release <a href='https://github.com/gkoh/furble/releases/tag/v3.0.0-rc1'>v3.0.0-rc1</a>.</em>

**Comment by gkoh** [comment]
After a significant amount of effort I am pleased to pre-release furble v3.0 with a shiny new user interface.
I have endeavoured to test on multiple devices and targets, however this effort is not infallible and I encourage users to raise any issues.

Functionally, there are very few changes, all previous features have been brought forward.
Unfortunately, the original M5StickC's display is now too small to offer a seamless experience, it is still supported, but some visual compromises have been made compared to newer controllers.

**Reply by matthudsonau** [reply]
Do you want a separate ticket for each issue, or just one big megathread?

**Reply by gkoh** [reply]
Separate tickets would be preferable.

> big megathread

Eeep, this makes it sound like I completely effed it ...

**Reply by matthudsonau** [reply]
Not at all, it's a massive improvement. Just a heap of little things (mostly buttons not playing nice)

**Reply by gkoh** [reply]
If it's a heap of related little things, I'm fine with a single/grouped ticket.

**Comment by gkoh** [comment]
Furthermore, the web installer is now on a new URL: https://web-installer.furble.net
The previous URL will continue to work for some time, but will eventually expire.

**Comment by gkoh** [comment]
v3.0.0-rc2 is now out.
It is a significant migration to using the ESP-IDF framework and introduces fairly significant power saving during normal use.

**Comment by gkoh** [comment]
v3.0.0-rc3 is now out addressing a bunch of minor UI issues.
This also adds a screen lock function for the M5Core2 (or any M5 device with touch and a PWR button).

**Comment by gkoh** [comment]
v3.0.0-rc4 is now out with support for the new v1.1 GPS unit.
UI for M5Core2 is further tweaked.

**Comment by gkoh** [comment]
v3.0.0-rc5 is now released with a handful of minor updates.
I expect v3.0.0 proper is not too far away.
